content/en/api/service_level_objectives/service_level_objectives_delete.md

```diff
@@ -9,6 +9,10 @@ external_redirect: /api/#delete-a-service-level-objective
 
 Permanently delete a SLO.
 
+If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint will return a 409 conflict error because the SLO is referenced in a dashboard.
```

Comment:
```suggestion
If an SLO is used in a dashboard, the DELETE /v1/slo/ endpoint returns a 409 conflict error because the SLO is referenced in a dashboard.
```